docs/.vuepress/plugins/assets-versioning/index.js

```diff
@@ -38,6 +40,19 @@ module.exports = (options, context) => {
       if ($page.currentVersion !== $page.latestVersion && $page.frontmatter.canonicalUrl) {
         $page.frontmatter.canonicalUrl = `https://handsontable.com/docs${$page.frontmatter.canonicalUrl}`;
       }
-    }
+    },
+
+    chainWebpack(config) {
+      const files = helpers.getVersions().map(version => ({
```

Comment:
getVersion needs BUILD_MODE argument. In another way, it will apply the `next` version for production env.

Reply:
Fixed within 2e6633e8e6fcb5828e68c9d95a2a2c6c9e37381c.